kintex7/mapping/resources.yaml

```diff
@@ -0,0 +1,10 @@
+# part number to pins
```

Comment:
I believe `resources.yml` should be kept in `prjxray`, as users don't need it, and it will not be auto-generated.  I suggest you keep it in `prjxray/settings/<family>/resources.yml`